### ISSUE TITLE
Set correct namelist.input for metonly

### ISSUE THREAD

**Issue by zxdawn** [CohenBerkeleyLab/AutoWRFChem-Base#44]
https://github.com/CohenBerkeleyLab/AutoWRFChem-Base/blob/9c1e34f08e4763d6fda082d23a3d85939147c9a4/prepinpt#L185-L204

Because you want to run real.exe with only the meterology, you should add these before `./real.exe`:

```
            if [[ $FDDA_ON == 1 ]]; then
                fdda_args="$pyargs --grid_fdda=0"
            else
                fdda_args=""
            fi  

            python $pyprog tempmod --emiss_opt=0 --bio_emiss_opt=0 --restart=.false. $fdda_args
```

**Comment by joshua-laughner** [comment]
Why? What if you want to run just WRF with nudging? Then you would need to have FDDA on for that run of real.exe to generate the FDDA file. The purpose of --met-only is not just to let you split up creating the met files and chem files in separate jobs, but to allow the automation to be used for pure-WRF jobs.

**Comment by zxdawn** [comment]
OK. If you want to run just WRF with nudging, it's right to turn FDDA on. But, you need to turn biogenic emissions off before `./real.exe`.

**Comment by zxdawn** [comment]
# Problem summary

* The function of running WRF-Chem with met-only doesn't work. `./autowrfchem prepinpt` works fine.

# Steps to reproduce

* What input data you are using (which met data, emissions data, etc.)
  NARR, NEI, MEGAN and MOZBC
* What Python version and package versions are being used
 Python3.6
* What compiler was used to compile WRF/WPS
  intel-compilers/15.0.1
* Version of netCDF C and Fortran libraries used for WRF
  netcdf/4.3.3.1/
* Specific commands to execute to cause the bug/crash
  `./autowrfchem prepinpt met-only`

# Logs
```
Pass: All raw met files found
Pass: WPS constants file found.
Running WPS...
Linking GEOGRID.TBL.ARW_CHEM as GEOGRID.TBL.
ln: creating symbolic link `GEOGRID.TBL.BACKUP': File exists
ln: creating symbolic link `Vtable.BACKUP': File exists
Running real.exe since only the meterology has been requested.
 starting wrf task            0  of            1
/WORK/nuist_chenq_2/xin/bug/A/BUILD/../WRFV3/run/wrfinput_d01 does not exist
```
rsl.error.0000:
```
 STARTING WITH BACKGROUND CHEMISTRY
 Subroutine input_chem_profile:
 READING MEGAN 2 EMISSIONS DATA
  mediation_integrate: med_read_wrf_chem_bioemissions: Open file wrfbiochemi_d01
 -------------- FATAL CALLED ---------------
 FATAL CALLED FROM FILE:  mediation_integrate.b  LINE:    2421
  med_read_wrf_chem_bioemissions: error opening wrfbiochemi_d01
 -------------------------------------------
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
```

# Reason and solution
It's necessary to turn biogenic emissions and anthropogenic emissions off before ./real.exe.

**Comment by joshua-laughner** [comment]
Please upload geogrid.log, ungrib.log, and metgrid.log from the BUILD/PREPLOGS directory as well.

**Comment by zxdawn** [comment]
Here's the attachment of log files.
[WPS_log.tar.gz](https://github.com/CohenBerkeleyLab/AutoWRFChem-Base/files/3123931/WPS_log.tar.gz)

**Comment by joshua-laughner** [comment]
Fixed in 364cda9
